### ISSUE TITLE
Title : Clock within the app.This clock must show the tasks (represented by some markers/flags) for the day. 

### ISSUE THREAD

**Issue by prajjwal7** [prajjwal7/Scheduler#1]

**Comment by ketan-b** [comment]
![Screenshot_20191230-170830](https://user-images.githubusercontent.com/54092325/72256863-427c6480-3630-11ea-8cae-bb7e48263372.jpg)

Will this work??

**Comment by ketan-b** [comment]
We will be further editing the panel colors according to the app's color scheme.

**Comment by thisHermit** [comment]
> ![Screenshot_20191230-170830](https://user-images.githubusercontent.com/54092325/72256863-427c6480-3630-11ea-8cae-bb7e48263372.jpg)
> 
> Will this work??

Looking cool

**Comment by prajjwal7** [comment]
> ![Screenshot_20191230-170830](https://user-images.githubusercontent.com/54092325/72256863-427c6480-3630-11ea-8cae-bb7e48263372.jpg)
> 
> Will this work??

This is not what we decided on, if I am not wrong.We wanted a circlular clock display with some markers for displaying the day's task..

**Comment by ketan-b** [comment]
Yeah but this can be an option because we are thinking of adding daily stats also, so to avoid clumsiness maybe we can use this otherwise circular is also there.
